e2e/support/component-webpack.config.js

```diff
@@ -81,16 +81,55 @@ module.exports = {
 };
 
 function resolveEmbeddingSdkPackage() {
-  let isEmbeddingSdkPackageInstalled = false;
-
   try {
-    const packagePath = require.resolve("@metabase/embedding-sdk-react");
-    if (packagePath.includes("node_modules")) {
-      isEmbeddingSdkPackageInstalled = true;
+    const nodeModulesPath = path.resolve(__dirname, "../../node_modules");
+
+    const sdkInNodeModulesPath = path.resolve(
+      nodeModulesPath,
+      SDK_PACKAGE_NAME,
+    );
+
+    // Try to resolve the package from the ancestor node_modules folder
+    const hasEmbeddingSdkInNodeModules = fs.existsSync(sdkInNodeModulesPath);
+
+    if (hasEmbeddingSdkInNodeModules) {
+      return {
+        isEmbeddingSdkPackageInstalled: true,
+        embeddingSdkPath: sdkInNodeModulesPath,
+      };
     }
-  } catch (err) {}
+
+    // Try to resolve the package with `require.resolve`
+    const requirePackagePath = require.resolve(SDK_PACKAGE_NAME);
+
+    if (requirePackagePath.includes("node_modules")) {
+      return {
+        isEmbeddingSdkPackageInstalled: true,
+        embeddingSdkPath: requirePackagePath,
+      };
+    }
```

Comment:
Using `require.resolve` was not working as it was resolving to `enterprise/frontend/src/embedding-sdk/index.ts`, instead of to `node_modules`. I tried specifying the module search path via `paths` and it still resolves to `index.ts` for some strange reason.

This is a fallback module resolution method in case the above fails. I think @deniskaber added this earlier so I don't want to remove it, but if the above method with `path.resolve` always works then we can remove this fallback.